socs/agents/ld_monitor/agent.py

```diff
@@ -0,0 +1,424 @@
+import argparse
+import os
+import socket
+import time
+
+import numpy
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+verbosity = False

```

Comment:
The `verbosity` argument is an okay approach for enabling debugging. An alternative would be to use debug logging. This would allow you to drop the tracking of `verbose`/`verbosity` and instead use:
```
self.log.debug('debug statement')
```
The advantage here is that the `LOGLEVEL` environment variable can be changed without modification of this `verbosity` constant in the code.

If you want to implement this check out the [Logging documentation](https://ocs.readthedocs.io/en/main/developer/agent_references/logging.html#). I wouldn't let it hold up the PR though.

Reply:
Removed verbosity constants and Implemented logging.